sdk/mysql/Azure.ResourceManager.MySql/api/Azure.ResourceManager.MySql.netstandard2.0.cs

```diff
@@ -230,19 +235,19 @@ public partial class MySqlPrivateLinkResourceData : Azure.ResourceManager.Models
         public MySqlPrivateLinkResourceData() { }
         public Azure.ResourceManager.MySql.Models.MySqlPrivateLinkResourceProperties Properties { get { throw null; } }
     }
-    public partial class QueryStatisticCollection : Azure.ResourceManager.ArmCollection
+    public partial class MySqlQueryStatisticCollection : Azure.ResourceManager.ArmCollection
     {
-        protected QueryStatisticCollection() { }
+        protected MySqlQueryStatisticCollection() { }
         public virtual Azure.Response<bool> Exists(string queryStatisticId, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response<bool>> ExistsAsync(string queryStatisticId, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual Azure.Response<Azure.ResourceManager.MySql.QueryStatisticResource> Get(string queryStatisticId, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual Azure.Pageable<Azure.ResourceManager.MySql.QueryStatisticResource> GetAll(Azure.ResourceManager.MySql.Models.TopQueryStatisticsInput input, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual Azure.AsyncPageable<Azure.ResourceManager.MySql.QueryStatisticResource> GetAllAsync(Azure.ResourceManager.MySql.Models.TopQueryStatisticsInput input, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.MySql.QueryStatisticResource>> GetAsync(string queryStatisticId, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.ResourceManager.MySql.MySqlQueryStatisticResource> Get(string queryStatisticId, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Pageable<Azure.ResourceManager.MySql.MySqlQueryStatisticResource> GetAll(Azure.ResourceManager.MySql.Models.MySqlTopQueryStatisticsInput input, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.AsyncPageable<Azure.ResourceManager.MySql.MySqlQueryStatisticResource> GetAllAsync(Azure.ResourceManager.MySql.Models.MySqlTopQueryStatisticsInput input, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
The list method here has a required parameter? Could you verify this?

Reply:
yes, based on swagger description: "The required parameters for retrieving top query statistics."